From 83158fa25f800058effa733f365e1ac650f4f1ff Mon Sep 17 00:00:00 2001
From: Wadamzmail <alksndrmetwakil@gmail.com>
Date: Tue, 9 Jun 2026 10:39:54 +0200
Subject: [PATCH] fix: update patches for latest termux-packages

---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index cfbec6e..85e648e 100644
--- a/build.sh
+++ b/build.sh
@@ -8,7 +8,7 @@ TERMUX_PKG_VERSION="2.46.0"
 TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL="https://mirrors.kernel.org/gnu/binutils/binutils-${TERMUX_PKG_VERSION}.tar.xz"
 TERMUX_PKG_SHA256=d75a94f4d73e7a4086f7513e67e439e8fcdcbb726ffe63f4661744e6256b2cf2
-TERMUX_PKG_DEPENDS="libc++, zlib, zstd"
+TERMUX_PKG_DEPENDS="zlib, zstd, libandroid-spawn, libandroid-glob"
 TERMUX_PKG_BREAKS="binutils (<< 2.46), binutils-bin, binutils-libs, binutils-dev"
 TERMUX_PKG_REPLACES="binutils (<< 2.46), binutils-bin, binutils-libs, binutils-dev"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
-- 
2.52.0

